modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -178,7 +147,6 @@ output application/json
     shokis: payload.root.users.*user map $.name
 }
 ----
-
 * If `com.mulesoft.dw.valueSelector.selectsAlwaysFirst` is set to `true`, the script returns the following output:
```

Comment:
```suggestion

* If `com.mulesoft.dw.valueSelector.selectsAlwaysFirst` is set to `true`, the script returns the following output:
```

Reply:
need to restore ALL blank lines above initial bullets so that the doc renders correctly in HTML

Reply:
fixed formatting manually